client/client.go

```diff
@@ -57,6 +57,9 @@ type Client interface {
 	// GetSecret retreaves preconfigred secret from specified store using key.
 	GetSecret(ctx context.Context, storeName, key string, meta map[string]string) (data map[string]string, err error)
 
+	// GetBulkSecret retreaves all preconfigred secrets for this application.
```

Comment:
nit: retrieves

Reply:
Hm, I ran make spell on this and for some reason didn't catch. On it.